api/serializers.py

```diff
@@ -143,6 +143,33 @@ def to_internal_value(self, data):
         return internals
 
 
+class AssetAssigneeSerializer(serializers.ModelSerializer):
+    assignee = serializers.SerializerMethodField()
+
+    class Meta:
+        model = AssetAssignee
+        fields = ("assignee",)
+
+    def get_assignee(self, obj):
+        if obj.user:
+            return {
```

Comment:
The returned response does not match the expected format. It ought to look something like this:
```json
{
    "count": 3,
    "next": null,
    "previous": null,
    "results": [
      {
        "id": 3,
        "assignee": "workspace_name"
      },
      {
        "id": 2,
        "assignee": "a.c@andela.com"
      },
      {
        "id": 1,
        "assignee": "department_name"
      }
    ]
}
```